google-cloud-bigtable/lib/google/cloud/bigtable/instance.rb

```diff
@@ -451,7 +451,7 @@ def tables
         # @param skip_lookup [Boolean] Create table object without verifying
         #   that the table resource exists.
         #   Calls made on this object will raise errors if the table.
-        #   does not exist. Default is `false`. Optional.
+        #   does not exist. Default value is `true`. Optional.
```

Comment:
If you want to flip the default behavior you might consider renaming the argument to something like `perform_lookup` so it can stay `nil` in the default case.

Reply:
Yes this a good idea.